realm/realm-jni/build.gradle

```diff
@@ -248,6 +248,67 @@ task deployCore(group: 'build setup', description: 'Deploy the latest version of
     }
 }
 
+task assembleRobolectric(description: "Build the Robolectric shared library for Darwin") {
+    def staticLibPath = "/usr/local/lib/librealm.a"
```

Comment:
It depends on `libdir` variable of `generic.mk` in the Realm Core library. I assumed most of us did not change this variable.